### PR TITLE
secure-boot.md: Move mbl-docs version to meta-mbl

### PR DESCRIPTION
**Pull request by jonathanhaigh-arm**
secure-boot.md started to be updated for inclusion in mbl-docs but that
never got finished. Move the partially updated document back here, along
with some changes due to partitioning changes and removal of dm-verity.

Also, adjust the image URLs to work in meta-mbl rather than the docs website.

For IOTMBL-531: Update website reference about partitions to match new
plans

### CODE REVIEW

File: docs/secure-boot.md [hunk 1]
```diff
@@ -1,183 +1,180 @@
-## Secure boot and firmware verification requirements
+# Secure boot and firmware verification requirements
 
-Copyright © 2018 Arm Limited.
+<!--note: this mustn't go live until the code does-->
```
Comment (jjohnson-arm) [inline docs/secure-boot.md]: This needs removing!
```suggestion
```

File: docs/secure-boot.md [hunk 2]
```diff
@@ -1,183 +1,180 @@
-## Secure boot and firmware verification requirements
+# Secure boot and firmware verification requirements
 
-Copyright © 2018 Arm Limited.
```
Comment (jjohnson-arm) [inline docs/secure-boot.md]: Please add this back and update it.
```suggestion
Copyright © 2018-2019 Arm Limited.
```

Comment (jonathanhaigh-arm) [reply docs/secure-boot.md]: Fixed in https://github.com/ARMmbed/meta-mbl/pull/637/commits/e25ca8a425be2a0dd065aef99369924edb291d02.

**Comment by jonathanhaigh-arm** [comment]
@jjohnson-arm, I've addressed your review comments - please take another look.

I'll cherry pick for 0.8 once this PR is merged.

**Comment by jonathanhaigh-arm** [comment]
PR for 0.8 backport: https://github.com/ARMmbed/meta-mbl/pull/644

**Pull request merged**